src/app/components/header/header.component.ts

```diff
@@ -1,9 +1,10 @@
 import { Component } from '@angular/core';
+import { NavbarComponent } from "../navbar/navbar.component";
 
 @Component({
   selector: 'app-header',
   standalone: true,
-  imports: [],
+  imports: [NavbarComponent],
```

Comment:
Navbar Component is separately created so, why need this header component ,
you can directly render the navbar component ?